### PR TITLE
🐛 make picture card tabbable only if action set

### LINKED ISSUE #4490: hui-picture-card always has tab index set
Home Assistant 104

Actions are optional for hui-picture-card but it will always set a tabindex as if it can handle your action.

https://github.com/home-assistant/home-assistant-polymer/blob/68d6faf4af96826dc4cd858ca0d29cffafe2f94f/src/panels/lovelace/cards/hui-picture-card.ts#L89

### PR DESCRIPTION
**Pull request by iantrich**
Fixes https://github.com/home-assistant/home-assistant-polymer/issues/4490

### CODE REVIEW

File: src/panels/lovelace/cards/hui-picture-card.ts [hunk 1]
```diff
@@ -86,10 +86,12 @@ export class HuiPictureCard extends LitElement implements LovelaceCard {
           hasHold: hasAction(this._config!.hold_action),
           hasDoubleClick: hasAction(this._config!.double_tap_action),
         })}
-        tabindex="0"
+        tabindex=${this._config.tap_action ? "0" : "-1"}
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-picture-card.ts]:
```suggestion
        . tabIndex=${this._config.tap_action ? "0" : "-1"}
```

Comment (bramkragten) [reply src/panels/lovelace/cards/hui-picture-card.ts]: `tab_action` can be `none` that should not give it a `tabindex`

Comment (iantrich) [reply src/panels/lovelace/cards/hui-picture-card.ts]: 👍 will use hasAction function

Comment (bramkragten) [reply src/panels/lovelace/cards/hui-picture-card.ts]: Or we go with `isDefined` as Paulus used.

Comment (iantrich) [reply src/panels/lovelace/cards/hui-picture-card.ts]: Yeah, no reason to set the attribute at all it not necessary 👍

File: src/panels/lovelace/cards/hui-picture-card.ts [hunk 2]
```diff
@@ -86,10 +87,14 @@ export class HuiPictureCard extends LitElement implements LovelaceCard {
           hasHold: hasAction(this._config!.hold_action),
           hasDoubleClick: hasAction(this._config!.double_tap_action),
         })}
-        tabindex="0"
+        tabindex=${ifDefined(hasAction(this._config.tap_action))
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-picture-card.ts]: tabindex=${ifDefined(hasAction(this._config.tap_action)
          ? "0"
          : undefined)}

**Review by bramkragten — APPROVED** [review]

**Pull request merged**